zerver/openapi/zulip.yaml

```diff
@@ -10829,6 +10832,7 @@ paths:
                                 "id": "1",
                                 "name": "green_tick",
                                 "source_url": "/user_avatars/1/emoji/images/1.png",
+                                "still_url": "",
```

Comment:
Let's extend these example responses to contain a second emoji with a non-empty `still_url`.